src/pages/events/consume-events.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Consume Events
+description: Learn how to consume events sent from Adobe Commerce to Adobe I/O Events.
+keywords:
+  - Events
+  - Extensibility
+---
+
+Events sent from Adobe Commerce to Adobe I/O events can be consumed in a few different ways. The following options for consuming events can be configured when adding events to your App Builder project and creating an event registration:
+
+* [Using the Journaling API](#using-the-journaling-api)
+* [Using a Webhook URL](#using-a-webhook-url)
+* [Using a Runtime Action](#using-a-runtime-action)
+* [Using Amazon EventBridge](#using-amazon-eventbridge)
+
+## Using the Journaling API
+
+When an Adobe I/O event registration is created, the subscribed events will by default be added to an ordered list, referred to as the journal. These events can be consumed using a journaling endpoint URL that is unique to the registration. For more information on reading events from the journal, see the [Introduction to Journaling](https://developer.adobe.com/events/docs/guides/journaling_intro/).
+
+Events can also be consumed from the Journaling API using the [Adobe I/O Events SDK](https://github.com/adobe/aio-lib-events). See [Subscribe to Events Using Journaling](https://developer.adobe.com/events/docs/guides/sdk/sdk_journaling/) for details and sample code.
+
+## Using a Webhook URL
+
+When creating or editing an event registration, a webhook URL can be registered. Doing so will cause subscribed Commerce events sent to Adobe I/O Events to be forwarded to the specified webhook URL. See [Introduction to Adobe I/O Events Webhooks](https://developer.adobe.com/events/docs/guides/) for more details.
```

Comment:
Can we add a bit more context like this "This option is useful for use cases that do not require App Builder to handle transformations to the event structure or additional API calls. This method of consuming events allows for easily forwarding events from Adobe Commerce to a 3rd party system via App Builder. Note that the destination system must have functionality to support a webhook URL endpoint in order to use this option." 

I think a screenshot of the webhook event registration UI would be useful here as well

Reply:
@sarahxxu do you think it would be helpful to include a note like "Be careful not to confuse this method of consuming events with [Runtime Actions as a Webhook] (https://developer.adobe.com/events/docs/guides/runtime_webhooks/#benefits-of-using-runtime-action-as-webhook). Runtime Actions as a Webhook is a functionality for creating a URL for App Builder to receive events from a 3rd party system. 

Personally I got very confused by this initially

Reply:
I've added this statement as an inline warning within this section for now